spec/unit/target/pod_target_spec.rb

```diff
@@ -631,14 +642,8 @@ module Pod
           scoped_pod_target.first.dependent_targets.first.name.should == 'OrangeFramework-Pods'
           scoped_pod_target.first.test_dependent_targets_by_spec_name.count.should == 1
           scoped_pod_target.first.test_dependent_targets_by_spec_name['OrangeFramework'].first.name.should == 'matryoshka-Pods'
-        end
-
-        it 'scopes app dependencies' do
```

Comment:
this was already fixed on master. I only took a tiny part of the 1-7-stable fix and merged the two. The same test is covered by the one above so its all good.